third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -23,17 +23,16 @@
 
 namespace xla {
 
-namespace {
-
-std::string PjRtDeviceToString(PjRtDevice* const device) {
+std::string PjRtComputationClient::PjRtDeviceToString(PjRtDevice* const device) const {
```

Comment:
Why are we removing the unnamed namespace? Are the functions here used elsewhere?